pkg/cloudprovider/providers/gce/gce_disks_test.go

```diff
@@ -80,7 +88,15 @@ func TestCreateDisk_Basic(t *testing.T) {
 func TestCreateDisk_DiskAlreadyExists(t *testing.T) {
 	/* Arrange */
 	fakeManager := newFakeManager()
-	gce := GCECloud{manager: fakeManager, managedZones: []string{"zone1"}}
+	alphaFeatureGate, featureGateErr := NewAlphaFeatureGate([]string{})
```

Comment:
Would it be beneficial to have a common init/setup routine?

Reply:
This is strictly for the calls made by the E2E test framework (independent of Kubernetes), and is configuration information. So beyond the existing shared code, no.